src/configuration.h

```diff
@@ -19,17 +19,13 @@
 #include <mutex>
 #include <string>
 
-namespace google {
+int main(int ac, char** av);
```

Comment:
You shouldn't need to pre-declare it here.

Reply:
I ran into a compilation issue without this forward declaration.  Specifically: `‘int main(int, char**)’ should have been declared inside ‘::’`

Reply:
Yeah, you're right, it needs to know about the `main` in the global namespace.